#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_time.h.dpatch by Steve Kemp <skx@debian.org>
##
## DP: Include <time.h> to fix segfault on some architectures.
## DP: Closes #315969

@DPATCH@
diff -urNad --exclude=CVS --exclude=.svn ./msgsnarf.c /tmp/dpep-work.9jQyoT/dsniff-2.4b1/msgsnarf.c
--- ./msgsnarf.c	2001-03-15 08:33:04.000000000 +0000
+++ /tmp/dpep-work.9jQyoT/dsniff-2.4b1/msgsnarf.c	2005-07-11 20:15:50.000000000 +0000
@@ -23,6 +23,7 @@
 #include <nids.h>
 #include <pcap.h>
 #include <pcaputil.h>
+#include <time.h>
 
 #include "buf.h"
 #include "decode.h"
diff -urNad --exclude=CVS --exclude=.svn ./sshow.c /tmp/dpep-work.9jQyoT/dsniff-2.4b1/sshow.c
--- ./sshow.c	2005-07-11 20:14:19.000000000 +0000
+++ /tmp/dpep-work.9jQyoT/dsniff-2.4b1/sshow.c	2005-07-11 20:15:26.000000000 +0000
@@ -15,6 +15,7 @@
 
 #include <sys/types.h>
 #include <sys/times.h>
+#include <time.h>
 
 #include <netinet/in_systm.h>
 #include <netinet/in.h>
